src/fn/watch-project.js

```diff
@@ -125,6 +130,44 @@ const processAppFormMedia = (formMediaDir, fileName) => {
     return false;
 };
 
+const processTrainingForm = (eventType, fileName) => {
```

Comment:
I'm wondering if we can reduce some of this code duplication. 
It looks like the only difference between `processTrainingForm` and `processAppForm` are the `validate???Forms`, `upload???Forms` and `upload???Forms`. Is it possible we could converge this code into a single function that knows which functions to call based on the type of form it is?

Reply:
I've made a refactor